README.md

```diff
@@ -75,7 +73,7 @@ systems with no GPU support, you should add the following dependencies:
 </dependency>
 <dependency>
   <groupId>org.tensorflow</groupId>
-  <artifactId>tensorflow-core-api</artifactId>
+  <artifactId>tensorflow-core-native</artifactId>
```

Comment:
Should we rev the version to 0.6.0 in these snippets?

Reply:
0.5.0 is still our last release, so we should stick to it. When I talk about snapshots a bit below in the README page, I show an example with 0.6.0-SNAPSHOT. Mentioning 0.6.0 would be misleading since it does not exists.